src/eggd_qc_to_workbooks.sh

```
@@ -63,9 +71,43 @@ main() {
+            wait
+        fi
+    done < <(find /home/dnanexus/mosdepth_inputs -name "*_markdup.per-base.bed.gz" -print0)
+
+    wait
+
     echo "running python"
 
-    python3 annotate_workbooks/annotate_workbooks_with_QC.py --multiqc_folder "$multiqc_folder" --reports_folder "$reports_folder" --config_json "$cells_to_edit" --file_suffix "$file_suffix"
+    python3 annotate_workbooks/annotate_workbooks_with_QC.py --multiqc_folder "$multiqc_folder" \
+    --reports_folder "$reports_folder" --intersect_folder "$intersect_folder" \
+    --config_json "$cells_to_edit" --file_suffix "$file_suffix"
```

Shellcheck (0.11.0)

[warning] file_suffix is referenced but not assigned.

(SC2154)